### ISSUE TITLE
json output format?

### ISSUE THREAD

**Issue by juju4** [volanja/ansible_spec#81]
Hello,

Could you support different output format, most notably json?
It would permit integration with other tools like monitoring as shown here
http://www.slideshare.net/m_richardson/serverspec-and-sensu-testing-and-monitoring-collide

Thanks

**Comment by juju4** [comment]
edit of .rspec with "--format json" is ok but maybe a command line argument would be nice

**Comment by volanja** [comment]
Hi

Please try it.
It might be that you want.

```
$ rake serverspec:Ansible-Sample-TDD SPEC_OPTS="--format json" 
```

**Comment by juju4** [comment]
Yes, that is an option.
But my problem currently is output of command is not a full json.
it's an informative message
```
Run serverspec for ...
... rspec --pattern roles/...
```
and a per host json.

so how to disable first part and consolidate everything in one json?
or maybe request testing only per host?

**Comment by juju4** [comment]
yes tail can leave header but what about per host json or aggregated valid one?
for example
```
$ rake serververspec:Ansible-Sample all
$ rake serververspec:Ansible-Sample host1
```
and all generating a full json output

Thanks

**Comment by volanja** [comment]
Hmm, ansible_spec can parse config of ansible, and provide serverspec parsed information.
So, I think that this problem caused by serverspec.

**Comment by volanja** [comment]
This issue has been marked as stale because it has not had recent activity.
It will be closed if no further activity occurs.
Thank you for your contributions.
